docs/kpod-diff.1.md

```diff
@@ -0,0 +1,45 @@
+% kpod(1) kpod-diff - Inspect changes on a container, layer, or image's filesystem
+% Dan Walsh
+# kpod-diff "1" "August 2017" "kpod"
+
+## NAME
+kpod diff - Inspect changes on a container, layer, or image's filesystem
+
+## SYNOPSIS
+**kpod** **diff** [*options* [...]] NAME
+
+## DESCRIPTION
+Displays changes on a container, layer, or image's filesystem.  The layer (or top layer of container/image) will be compared to its parent layer
```

Comment:
Way too complicated.  User have no idea what a layer is, so don't mention it.
docker diff man page says.

```       Inspect changes on a container's filesystem. You can use the full or shortened container ID or the container name set using docker run --name option.
```